<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>520</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>clock</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_register" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>351</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>270</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>年龄:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>320</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>性别:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>327</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>男</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>327</y>
       <width>73</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>女</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_register_message">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>550</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_register_login">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>570</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>已有账号？点我登陆</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_register_top" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>121</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_login_min_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>-10</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_login_quit_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_register_welcome">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>30</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_register_wel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>体重:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>身高:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="maximum">
       <double>2.300000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(kg)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(m)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_register_effect" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>331</width>
       <height>431</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_register">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>261</width>
        <height>201</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="lineEdit_register_name">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_username">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_password">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <zorder>widget_2</zorder>
     <zorder>pushButton_register</zorder>
    </widget>
    <zorder>widget_register_effect</zorder>
    <zorder>widget</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>spinBox</zorder>
    <zorder>checkBox</zorder>
    <zorder>checkBox_2</zorder>
    <zorder>label_register_message</zorder>
    <zorder>label_register_login</zorder>
    <zorder>widget_register_top</zorder>
   </widget>
   <widget class="QWidget" name="widget_login" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>421</width>
      <height>331</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_login">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑 Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_login_username">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_login_password">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_login_register">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>注册账号</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>121</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_login_min">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>-10</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_login_quit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_login_title">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>171</x>
       <y>71</y>
       <width>74</width>
       <height>74</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_login_message">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>290</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_headphoto">
     <property name="geometry">
      <rect>
       <x>173</x>
       <y>73</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
